packages/components/src/templates/next/components/internal/Navbar/Navbar.tsx

```diff
@@ -6,7 +6,7 @@ import { useResizeObserver } from "usehooks-ts"
 
 import type { NavbarProps } from "~/interfaces"
 import { tv } from "~/lib/tv"
-import { isExternalUrl } from "~/utils"
+import { getReferenceLinkHref, isExternalUrl } from "~/utils"
```

Comment:
not sure if bug but i expected to see `getReferenceLinkHref` being used but it isn't

Reply:
ah ok it got shifted into navbar item